Model/Logger.php

```diff
@@ -140,6 +147,10 @@ public function log($message, $label = '')
                     $label = '[' . strtoupper($label) . '] ';
                 }
 
+                if ($this->taxjarConfig->isSandboxEnabled()) {
+                    $label .= '[SANDBOX] ';
```

Comment:
@ryanreeves-taxjar Can you provide sample output from the logs here?

Reply:
![sandbox_logs](https://user-images.githubusercontent.com/44789510/85592643-95a50b00-b603-11ea-9a6c-535230ab73c1.png)

Reply:
@ryanreeves-taxjar Can we have `[SANDBOX]` before `[POST]` and `[API]` so it lines up and makes it easier to scan?

Reply:
👍